### PR TITLE
templates: CMS: addition to troubleshooting

### PR DESCRIPTION
**Pull request by atrisovic**
* Closes: #1234

Signed-off-by: Ana Trisovic <ana.trisovic@gmail.com>

### CODE REVIEW

**Comment by atrisovic** [comment]
Ok! 👍🏻 👌🏻

File: cernopendata/modules/fixtures/data/articles/cms-guide-troubleshooting/errors.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
+## Here is an initial list of frequently asked questions:
+
+- The root password for CMS Open Data VM is `password`
+
+- In some versions of VirtualBox, the CMS Open Data VM does not open correctly. This has happened for example for VirtulBox 5.0.32, but for example 5.1.8 is OK.
+
+- xml files do not download correctly by clicking on "Download" e.g. `http://opendata.cern.ch/record/560/files/BuildFile.xml` in [record 560](/record/560)
```
[the remark this answers is not in the document]
Comment (katilp) [reply cernopendata/modules/fixtures/data/articles/cms-guide-troubleshooting/errors.md]: whether true or not, the files are easiest downloaded to VM with wget so we may want to mention it in any case

**Comment by atrisovic** [comment]
ok this is now fixed 👍

**Comment by ioannistsanaktsidis** [comment]
@atrisovic  please rebase against latest master and I will merge.

**Comment by atrisovic** [comment]
done! ✅

**Comment by ioannistsanaktsidis** [comment]
@tiborsimko  so  we finally remove the about static pages and we keep only the records one ? Please let me know so as @atrisovic  to adapt this PR.

**Pull request merged**